patches/minecraft/net/minecraft/client/util/RecipeBookCategories.java.patch

```diff
@@ -0,0 +1,188 @@
+--- a/net/minecraft/client/util/RecipeBookCategories.java
++++ b/net/minecraft/client/util/RecipeBookCategories.java
+@@ -9,7 +9,7 @@
+ import net.minecraftforge.api.distmarker.OnlyIn;
+ 
+ @OnlyIn(Dist.CLIENT)
+-public enum RecipeBookCategories {
++public enum RecipeBookCategories implements net.minecraftforge.common.extensions.IForgeRecipeBookCategory<net.minecraft.item.crafting.IRecipeType<?>> {
+    SEARCH(new ItemStack(Items.field_151111_aL)),
+    BUILDING_BLOCKS(new ItemStack(Blocks.field_196584_bK)),
+    REDSTONE(new ItemStack(Items.field_151137_ax)),
+@@ -36,4 +36,176 @@
+    public List<ItemStack> func_202903_a() {
+       return this.field_202904_j;
+    }
++
++   //--FORGE
```

Comment:
This is not correct, we use:
`/*================================ FORGE START ================================================*/`
as a separator,

You should probably add this on all files where you add a footer section with just forge changes.
Additionally it also needs an end marker:
`/*================================ FORGE END ================================================*/`

Reply:
Replaced with valid forge separator